debian/control

```diff
@@ -2,13 +2,12 @@ Source: ralph
 Section: python
 Priority: extra
 Maintainer: pylabs@allegrogroup.com
-Build-Depends: debhelper (>= 9), python
```

Comment:
That's why it is on Build-Depends probably? When the user install it, it will not be included, right?